src/portal-components/NavUserLink.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import {UserProfile} from "synapse-react-client/dist/utils/synapseTypes";
+import UserCard from "synapse-react-client/dist/containers/UserCard";
+import {SynapseConstants} from "synapse-react-client";
+import SvgIcon from "@material-ui/core/SvgIcon";
+
+type NavUserLinkProps = {
+  userProfile: UserProfile | undefined
+}
+
+const NavUserLink: React.FunctionComponent<NavUserLinkProps> = ({
+  userProfile,
+}) => {
+
+  return(
+    <>
+      <UserCard
+        userProfile={userProfile}
+        size={SynapseConstants.SMALL_USER_CARD}
+        preSignedURL={userProfile?.clientPreSignedURL}
+        hideText={true}
+        link="javascript:void(0)"
+      />
+      <SvgIcon className="arrow-down">
```

Comment:
Consider pulling in material ui icons dependency for definition.  Or, if that's too heavy-handed, refactor this arrow-down icon into it's own component for future reuse.

Reply:
I have updated the code to pull the icon directly from the material ui icons package.